### PR TITLE
Add Time Zone

### PR DESCRIPTION
**Pull request by olo-dw**
I propose to add the Time Zone parameter with "UTC" as a default value.

~~Note: I added the PR #30 in order to avoid the issue #29 .~~

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -34,7 +34,7 @@ script:
 
   # Make sure GitLab is running.
   - >
-    curl --insecure -s -o /dev/null -w "%{http_code}" https://localhost/users/sign_in
+    curl --insecure -s -o /dev/null -w "%{http_code}" https://localhost/users/password/new
```
Comment (geerlingguy) [inline .travis.yml]: Can you rebase against new master without the /password/new commit?

**Comment by olo-dw** [comment]
I did rebase against the new master without the /password/new commit 4aeaec0.

**Comment by geerlingguy** [comment]
Same here - can you fix the conflict? Maybe just do on one of your two remaining PRs and I'll merge that, then fix the last one and I'll merge that :)

**Comment by olo-dw** [comment]
I fixed the conflict.

**Pull request merged**